src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,36 @@
+package duke;
```

Comment:
I dont think file naming of packages should be capitalised? So it should be `src/main/java/duke/Duke.java` instead of `src/main/java/Duke/Duke.java`.